model_analyzer/result/model_config_measurement.py

```diff
@@ -285,8 +286,8 @@ def _calculate_weighted_score(self, other):
                 return 0
 
             metric_diff = self_metric - other_metric
-            weighted_score += (weight *
-                               (metric_diff.value() / self_metric.value()))
+            weighted_score += (weight * (metric_diff.value() / mean(
+                [self_metric.value(), other_metric.value()])))
```

Comment:
this line is so hard to parse. I know what it is supposed to be doing and I still can't manage to line up all the brackets and parenthesis in my head.  Can you refactor it a bit? Maybe pull the mean code out just like `metric_diff` was?